### PR TITLE
Edit for content audit

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes

- https://docs.docker.com/ee/ucp/admin/configure/configure-rbac-kube/
- https://docs.docker.com/ee/ucp/admin/configure/enable-audit-logging/
- https://docs.docker.com/ee/ucp/admin/configure/run-only-the-images-you-trust/
- https://docs.docker.com/ee/ucp/authorization/create-teams-with-ldap/

### Related issues (optional)
- Updating topics to prepare for content migration
- https://docs.google.com/spreadsheets/d/1cGlW5jJoEHf3PI3oKd-DmrefdUrCmNpxJUiZQEg1qes/edit#gid=0

### CODE REVIEW

**Comment by traci-morrison** [comment]
@Dawn-Wood @KoryKessel-Mirantis PTAL

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 1]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 
 ## Creating role grants
 
-Kubernetes provides 2 types of role grants:
+Kubernetes provides two types of role grants, **ClusterRoleBinding** and **RoleBinding**:
 
-- `ClusterRoleBinding` which applies to all namespaces
-- `RoleBinding` which applies to a specific namespace
+- **ClusterRoleBinding** which applies to all namespaces
+- **RoleBinding** which applies to a specific namespace
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: I suggest foregoing the short bullet list and instead bringing the small detail for each role grant type up into the lead-in sentence:

"Kubernetes provides two types of role grants, **ClusterRoleBinding** (which applies to all namespaces) and **RoleBinding** (which applies to a specific namespace)."

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 2]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 
-Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all the roles for a particular cluster under **Access Control** then **Roles**. Select Kubernetes or Swarm to view the specific roles for each.
+Starting with UCP 3.1, Kubernetes and Swarm roles have separate views. You can view all of the roles for a particular cluster under **Access Control** > **Roles**. Select **Kubernetes** or **Swarm** to view the specific roles for each.
 
 ## Creating roles
 
-You create Kubernetes roles either through the CLI using `kubectl` or through the UCP web interface.
-
-To create a Kubernetes role in the UCP web interface:
+You can create Kubernetes roles either through the CLI using [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) or through the UCP web interface. The following procedure uses the UCP web interface.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: I suggest placing "either" after the first "through" and removing the second "through".

"You can create Kubernetes roles through either the CLI using [kubectl] or the UCP web interface (the procedure that follows)."

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 3]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 
 1. From the UCP UI, select **Access Control**.
 2. From the left navigation menu, select **Roles**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32roles.png)
 3. Select the **Kubernetes** tab at the top of the window.
 4. Select **Create** to create a Kubernetes role object in the following dialog:
     ![Kubernetes Role Creation in UCP](/ee/ucp/images/kube-role-create.png)
-5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a `ClusterRole` where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
+5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a ClusterRole where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
 6. Provide the YAML for the role, either by entering it in the **Object YAML** editor or select **Click to upload a .yml file** to choose and upload a .yml file instead.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Change 'but' to 'however'.

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 4]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
+You can create Kubernetes roles either through the CLI using [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) or through the UCP web interface. The following procedure uses the UCP web interface.
 
 1. From the UCP UI, select **Access Control**.
 2. From the left navigation menu, select **Roles**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32roles.png)
 3. Select the **Kubernetes** tab at the top of the window.
 4. Select **Create** to create a Kubernetes role object in the following dialog:
     ![Kubernetes Role Creation in UCP](/ee/ucp/images/kube-role-create.png)
-5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a `ClusterRole` where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
+5. Select a namespace from the **Namespace** drop-down list. Selecting a specific namespace creates a role for use in that namespace, but selecting all namespaces creates a ClusterRole where you can create rules for cluster-scoped Kubernetes resources as well as namespaced resources.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Change 'but' to 'however'.

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 5]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
+Kubernetes provides two types of role grants, **ClusterRoleBinding** and **RoleBinding**:
 
-- `ClusterRoleBinding` which applies to all namespaces
-- `RoleBinding` which applies to a specific namespace
+- **ClusterRoleBinding** which applies to all namespaces
+- **RoleBinding** which applies to a specific namespace
 
 To create a grant for a Kubernetes role in the UCP web interface:
 
 1. From the UCP UI, select **Access Control**.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Remove "From the UCP UI, " (covered just one line above).

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 6]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 2. From the left navigation menu, select  **Grants**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32grants.png)
 3. Select the **Kubernetes** tab at the top of the window. All grants to Kubernetes roles can be viewed in the Kubernetes tab.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Edit into one sentence, "Select the **Kubernetes** tab at the top of the window to view all of the grants available to Kubernetes roles."

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 7]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 6. To create a user role binding, select a username from the **Users** drop-down list then select **Next**.
 7. Select a resource set for the subject. The **default** namespace is automatically selected. To use a different namespace, select the **Select Namespace** button next to the desired namespace. For `Cluster Role Binding`, slide the **Apply Role Binding to all namespaces** selector to the right.
     ![Kubernetes Create User Role Binding in UCP](/ee/ucp/images/kube-grant-rolebinding.png)
-8. Select **Next** to continue.
-9. Select the **Cluster Role** from the drop-down list. If you create a `ClusterRoleBinding` (by selecting **Apply Role Binding to all namespaces**) then you may only select ClusterRoles. If you select a specific namespace, you can choose any role from that namespace or any ClusterRole.
+8. Select **Next**.
+9. Select the **Cluster Role** from the drop-down list. If you create a ClusterRoleBinding (by selecting **Apply Role Binding to all namespaces**), then you may only choose ClusterRoles. If you select a specific namespace, you can choose any role from that namespace or any ClusterRole.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Suggest editing the second sentence to read:

"If you select **Apply Roles Binding to all namespaces** to create a ClusterRolebinding, only *ClusterRoles* can be selected. If, though, you select a specific namespace you can choose any role from that namespace or any ClusterRole."

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 8]
```diff
@@ -1,99 +1,95 @@
...
+individual users, administrators, or software components that have affected the 
 system. They are focused on external user/agent actions and security rather than 
 understanding state or events of the system itself.
 
 Audit logs capture all HTTP actions (GET, PUT, POST, PATCH, DELETE) to all UCP 
-API, Swarm API and Kubernetes API endpoints that are invoked (except for the 
-ignored list) and sent to Docker Engine via stdout. Creating audit logs is a UCP 
-component that integrates with Swarm, Kubernetes, and UCP APIs.
+API, Swarm API, and Kubernetes API endpoints that are invoked (except for the 
+ignored list) and sent to Docker Engine through standard output (stdout). Creating audit logs is a UCP component that integrates with Swarm, Kubernetes, and UCP APIs.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: I suggest putting the last sentence into parenthesis at the end of the previous sentence.

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 9]
```diff
@@ -1,99 +1,95 @@
...
 
 ## Logging levels
 
-To allow more control to administrators over the audit logging, three audit 
-logging levels are provided:
+The following levels are provided to give administrators more control over audit logging:
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing "The following levels" to "Three audit logging", and removing "over audit logging".

For the bullet points that follow, I think the content would be better served in a table.

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 10]
```diff
@@ -1,99 +1,95 @@
...
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data. 
 
 ## Benefits
 
 You can use audit logs to help with the following use cases:
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest editing the sentence to read: "Audit logs can be of help with the following use cases:"

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 11]
```diff
@@ -1,99 +1,95 @@
...
 
-- **Chargeback** - You can use audit logs and information about the resources to 
+- **Chargeback:** You can use audit logs and information about the resources to 
 generate chargeback information.
 
-- **Alerting** - If there is a watch on an event stream or a notification 
+- **Alerting:** If there is a watch on an event stream or a notification 
 created by the event, alerting features can be built on top of event tools that 
-generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+generate alerts for Ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: In my opinion, the descriptions for the four benefit types are mostly superfluous. If you find that you agree, I would eliminate the bullet list and instead roll the benefits up into the first sentence:

"Audit logs can be of help with historical troubleshooting, security analysis and auditing, generating chargeback information, and creating useful alerts."

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 12]
```diff
@@ -1,99 +1,95 @@
 ---
 title: Enable audit logging on UCP
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest removing 'Enable' from the title (as the enabling is just one part of the content offered in the topic).

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 13]
```diff
@@ -1,99 +1,95 @@
...
 > container logs of the `ucp-controller` container on each UCP manager node. 
-> Please ensure you have a 
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data. 
 
 ## Benefits
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing the heading to read: "Benefits of Audit Logging"

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 14]
```diff
@@ -1,99 +1,95 @@
...
+- **Alerting:** If there is a watch on an event stream or a notification 
 created by the event, alerting features can be built on top of event tools that 
-generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+generate alerts for Ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
 
 ## Enabling UCP audit logging
 
-UCP audit logging can be enabled via the UCP web user interface, the UCP API or 
-via the UCP configuration file.
+UCP audit logging can be enabled through the UCP web user interface, UCP API, or UCP configuration file.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing the sentence to read: "Enable audit logging via either the UCP user interface, the UCP API, or the UCP configuration file."

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 15]
```diff
@@ -1,99 +1,95 @@
...
 
 ### Enabling UCP audit logging using the web UI
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing "using" to "via".

File: ee/ucp/admin/configure/run-only-the-images-you-trust.md [hunk 16]
```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
...
+4. The IT operations team deploys a service. If the image used for the service was signed by the QA team, UCP deploys it. Otherwise, UCP refuses to deploy the image.
 
 ## Configure UCP
 
-To configure UCP to only allow running services that use Docker images you
-trust, go to the UCP web UI, navigate to the **Admin Settings** page, and in
-the left pane, click **Docker Content Trust**.
+To configure UCP to only allow running services that use Docker trusted images:
+
+1. From the UCP web UI, navigate to the **Admin Settings** page.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/run-only-the-images-you-trust.md]: Change "From" to "Using". Also, I don't think we need the word 'web' before 'UI' as there is no other UCP UI in play.

File: ee/ucp/admin/configure/configure-rbac-kube.md [hunk 17]
```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
...
 
-Kubernetes provides 2 types of role grants:
+Kubernetes provides two types of role grants, **ClusterRoleBinding** and **RoleBinding**:
 
-- `ClusterRoleBinding` which applies to all namespaces
-- `RoleBinding` which applies to a specific namespace
+- **ClusterRoleBinding** which applies to all namespaces
+- **RoleBinding** which applies to a specific namespace
 
 To create a grant for a Kubernetes role in the UCP web interface:
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/configure-rbac-kube.md]: Suggest changing 'web interface' to 'user interface'.

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 18]
```diff
@@ -1,99 +1,95 @@
...
 
-1. Log in to the **UCP** Web User Interface
-2. Navigate to **Admin Settings**
-3. Select **Audit Logs**
+1. Log in to the **UCP** web UI.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing "web UI" to "user interface".

File: ee/ucp/admin/configure/enable-audit-logging.md (hunk 15 above)
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest changing "using the web UI" to "through the user interface".

File: ee/ucp/admin/configure/run-only-the-images-you-trust.md [hunk 19]
```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
 
 >{% include enterprise_label_shortform.md %}
 
-With Docker Universal Control Plane you can enforce applications to only use
+With Docker Universal Control Plane (UCP), you can enforce applications to only use
 Docker images signed by UCP users you trust. When a user tries to deploy an
 application to the cluster, UCP checks if the application uses a Docker image
-that is not trusted, and won’t continue with the deployment if that’s the case.
+that is not trusted. If the image is not trusted, UCP will not continue with the deployment.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/run-only-the-images-you-trust.md]: Suggest editing the sentence to read: "Each time a user attempts to deploy an application to the cluster, UCP checks whether the application is using a trusted Docker image (and will halt the deployment if that is not the case)."

File: ee/ucp/admin/configure/run-only-the-images-you-trust.md [hunk 20]
```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
...
+2. In the left navigation pane, click **Docker Content Trust**.
+3. Select the **Run only signed images** option.
 
-Select the **Run Only Signed Images** option to only allow deploying
-applications if they use images you trust.
+    ![UCP settings](../../images/run-only-the-images-you-trust-2.png){: .with-border}
 
-![UCP settings](../../images/run-only-the-images-you-trust-2.png){: .with-border}
+    With this setting, UCP allows deploying any image as long as the image has
+    been signed. It doesn't matter who signed the image.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/run-only-the-images-you-trust.md]: In my opinion, the second sentence is superfluous.

File: ee/ucp/admin/configure/run-only-the-images-you-trust.md [hunk 21]
```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
...
 
-With this setting, UCP allows deploying any image as long as the image has
-been signed. It doesn't matter who signed the image.
+    To enforce that the image needs to be signed by specific teams, click **Add Team**
+    and select those teams from the list.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/run-only-the-images-you-trust.md]: Suggest editing to read: "To enforce the requirement that the image be signed by specific teams, click **Add Team** and select the pertinent teams from the list."

File: ee/ucp/admin/configure/run-only-the-images-you-trust.md [hunk 22]
```diff
@@ -6,55 +6,49 @@ keywords: ucp, dtr, security, trust
...
+    team, or by someone that is a member of all of those teams.
 
-If you specify multiple teams, the image needs to be signed by a member of each
-team, or someone that is a member of all those teams.
+4. Click **Save.** 
 
-Click **Save** for UCP to start enforcing the policy. From now on, existing
-services will continue running and can be restarted if needed, but UCP will only
-allow deploying new services that use a trusted image.
+    UCP starts enforcing the policy. From now on, existing services continue running and can be restarted if needed, but UCP only allows deploying new services that use a trusted image.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/run-only-the-images-you-trust.md]: Suggest editing to read: "At this point, UCP starts enforcing the policy. Existing services will continue running and can be restarted if needed, however UCP only allows the deployment of new services that use a trusted image."

File: ee/ucp/authorization/create-teams-with-ldap.md [hunk 23]
```diff
@@ -6,28 +6,25 @@ keywords: authorize, authentication, users, teams, UCP, LDAP
 
 >{% include enterprise_label_shortform.md %}
 
-To enable LDAP in UCP and sync to your LDAP directory:
+If Docker Enterprise is configured to sync users with your organization's Lightweight Directory Access Protocol (LDAP) directory server, you can enable syncing the new team's members when creating a new team or when modifying settings of an existing team.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/authorization/create-teams-with-ldap.md]: Suggest editing to read: "If Docker Enterprise is configured to sync users with a Lightweight Directory Access Protocol (LDAP) directory server, you can enable the syncing of new team members to occur either when creating a new team or when modifying settings of an existing team."

File: ee/ucp/authorization/create-teams-with-ldap.md [hunk 24]
```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
 ### Match Group Members (Direct Bind)
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/authorization/create-teams-with-ldap.md]: Suggest editing the first sentence to read: "The Direct Bind option specifies that team members be synced directly with member of a group in the organization's LDAP directory."

File: ee/ucp/authorization/create-teams-with-ldap.md [hunk 25]
```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
...
+- **Group DN:** The distinguished name (DN) of the group from which to select users.
+- **Group Member Attribute:** The value of this group attribute corresponds to
   the distinguished names of the members of the group.
 
-
 ### Match Search Results (Search Bind)
 
 This option specifies that team members should be synced using a search query
 against your organization's LDAP directory. The team's membership will be
 synced to match the users in the search results.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/authorization/create-teams-with-ldap.md]: Suggest editing the first sentence to read: "The Search Bind option specifies that team members be synced using a search query against the organization's LDAP directory."

File: ee/ucp/authorization/create-teams-with-ldap.md [hunk 26]
```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
...
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
 synced to match the membership of the group.
 
-- **Group DN**: The distinguished name of the group from which to select users.
-- **Group Member Attribute**: The value of this group attribute corresponds to
+- **Group DN:** The distinguished name (DN) of the group from which to select users.
+- **Group Member Attribute:** The value of this group attribute corresponds to
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/authorization/create-teams-with-ldap.md]: Would be great to have this information in a small table and not an overly-detailed bullet list.

File: ee/ucp/authorization/create-teams-with-ldap.md [hunk 27]
```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
...
 
-- **Search Base DN**: Distinguished name of the node in the directory tree where
+- **Search Base DN:** Distinguished name of the node in the directory tree where
   the search should start looking for users.
-- **Search Filter**: Filter to find users. If null, existing users in the search
+- **Search Filter:** Filter to find users. If null, existing users in the search
   scope are added as members of the team.
-- **Search subtree**: Defines search through the full LDAP tree, not just one
+- **Search subtree:** Defines search through the full LDAP tree, not just one
   level, starting at the Base DN.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/authorization/create-teams-with-ldap.md]: Would be great to have this information in a small table and not an overly-detailed bullet list.

File: ee/ucp/admin/configure/enable-audit-logging.md (hunk 9 above)
Comment (traci-morrison) [reply ee/ucp/admin/configure/enable-audit-logging.md]: Definition lists are not supported in GitHub Markdown.

**Comment by traci-morrison** [comment]
> Comments inline.

Changes incorporated. PTAL.

**Comment by traci-morrison** [comment]
> Comments inline.

Changes incorporated. PTAL.

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 28]
```diff
@@ -1,99 +1,78 @@
...
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
-
-## Benefits
-
-You can use audit logs to help with the following use cases:
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Suggest removing "start to".

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 29]
```diff
@@ -1,99 +1,78 @@
...
-via the UCP configuration file.
+Enable audit logging using either the UCP UI, the UCP API, or the UCP configuration file.
 
-### Enabling UCP audit logging using the web UI
+### Enabling UCP audit logging through the UI
 
-1. Log in to the **UCP** Web User Interface
-2. Navigate to **Admin Settings**
-3. Select **Audit Logs**
+1. Log in to the **UCP** UI.
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Should UCP be in bold here?

File: ee/ucp/admin/configure/enable-audit-logging.md [hunk 30]
```diff
@@ -1,99 +1,78 @@
...
+2. Navigate to **Admin Settings.**
```
Comment (KoryKessel-Mirantis) [inline ee/ucp/admin/configure/enable-audit-logging.md]: Meld Steps 1 and 2 together, as now seen in the following section, **Configure UCP**:

1. Access the UCP UI and browse to the **Admin Settings** page.

File: _data/toc.yaml [hunk 31]
```diff
@@ -1667,7 +1667,7 @@ manuals:
         - path: /ee/ucp/admin/configure/configure-rbac-kube/
           title: Configure native Kubernetes role-based access control
         - path: /ee/ucp/admin/configure/enable-audit-logging/
-          title: Create UCP audit logs
+          title: Enable audit logging on UCP
```
Comment (KoryKessel-Mirantis) [inline _data/toc.yaml]: Should this now read "Audit logging on UCP", to be in line with the change you made to the heading?

**Comment by traci-morrison** [comment]
> A few more inline comments, but mostly there.

Changes made. PTAL.

**Pull request merged**